modules/core/src/test/java/org/apache/ignite/testsuites/IgnitePdsTestSuite8.java

```diff
@@ -82,6 +83,7 @@
         GridTestUtils.addTestIfNeeded(suite, PageLockTrackerResourcesTest.class, ignoredTests);
 
         GridTestUtils.addTestIfNeeded(suite, IgnitePdsCacheEntriesExpirationTest.class, ignoredTests);
+        GridTestUtils.addTestIfNeeded(suite, ActivationOnExpirationTimeoutTest.class, ignoredTests);
```

Comment:
`ActivationOnExpirationTimeoutTest` uses atomic cache, why do we need MVCC test?

Reply:
Fixed